src/main/java/internetshop/controllers/cart/RemoveProductFromCartController.java

```diff
@@ -14,7 +14,6 @@
 
 @WebServlet("/cart/deleteProduct")
```

Comment:
```suggestion
@WebServlet("/cart/products/delete")
```
краще щоб посилання мали одинакову структуру